tools/stress/src/org/apache/cassandra/stress/StressProfile.java

```diff
@@ -667,6 +670,19 @@ public SchemaInsert getInsert(Timer timer, PartitionGenerator generator, SeedMan
         return builder.apply(defValue);
     }
 
+    private static boolean isTypeSupported(com.datastax.driver.core.DataType dataType) {
+        if (!dataType.isCollection())
+            return true;
+        if (dataType.getName().toString().toUpperCase() == "MAP")
```

Comment:
Does this work for you? I think it's cleaner than figuring out it by the name.

```suggestion
        if (dataType instanceof MapType)
```

Reply:
What is that single class you are referring to? The `DataTypes` class in the Java driver instantiates different classes depending on the type. The `DefaultMapType` class implements that `MapType` interface.

Reply:
Pekka, you are looking at 4.x branch of either datastax driver or scylla driver , but we are currently on:
          <dependency groupId="com.scylladb" artifactId="scylla-driver-core" version="3.7.1-scylla-2" classifier="shaded" />
There is no such classes there: https://github.com/scylladb/java-driver/blob/3.7.1-scylla/driver-core/src/main/java/com/datastax/driver/core/DataType.java
.
Also it does not matter anymore since this part of the code was changed to accommodate all cases.
Anyway thanks for tip, i've discovered corner case due to it.